### ISSUE TITLE
Provide a template repo, or document how to set one up

### ISSUE THREAD

**Issue by d3fault** [probot/dco#166]
This DCO bot is useful and neat but currently all it does is force the user to provide the -s flag to their commit command....... without clearly stating that doing so is going to be interpreted as an Agreement to the DCO (I submitted a PR to be more clear about it: #165). Indeed even the git-commit manpages state that the Sign-Off  line bears no meaning *unless* a project assigns it meaning: "The meaning of a signoff depends on the project" (man git-commit). The mention of DCO in the next sentence in the manpages should be ignored since it's preceded by the non-legally-binding word "typically".

So basically a barebones/minimal template repo might look like this:
```
./copying
./readme.md <-- references copying and contributing.md and has a lorem ipsum description
./contributing.md <-- uses wording like "by submitting a pull request, you Agree to the DCO" (or maybe just "by contributing"?)

```

I actually am unsure of what the template repo should look like, hence this issue instead of a PR
